lib/gui/elistboxcontent.cpp

```diff
@@ -959,11 +959,16 @@ int eListboxPythonMultiContent::getMaxItemTextWidth()
 				{
 				case TYPE_TEXT: // text
 				{
-					ePyObject px = PyTuple_GET_ITEM(item, 1), pstring = PyTuple_GET_ITEM(item, 7);
+					ePyObject px = PyTuple_GET_ITEM(item, 1), pfnt = PyTuple_GET_ITEM(item, 5), pstring = PyTuple_GET_ITEM(item, 7);
 
 					if (PyLong_Check(pstring) && data) /* if the string is in fact a number, it refers to the 'data' list. */
 						pstring = PyTuple_GetItem(data, PyLong_AsLong(pstring));
 
+					if (pfnt) {
+						int fnt_i = PyLong_AsLong(pfnt);
+						fnt = m_font[fnt_i];
```

Comment:
Sorry, chack is needed before this.

Reply:
hmm what is the point of that? if its supposed to check is there given index in the array then the check should be only if there is missing index...
In my opinion that can not happen however since if that is the case it will crash much before that function to be executed.

But well a guard check can be added anyway.